README.md

```diff
@@ -1,31 +1,28 @@
 # SheSharp Knowledge Hub
 
-## Introduction
-
 This is a collection of resources gathered by SheSharp community.
 
 It includes a table of contents for you to navigate to the sections that matter the most to you.
 
-The resources collected here cover different areas, such as software development, data analysis, product management and quality assurance.
-We have made an effort to include as many free resources as possible.
+
+The resources collected here cover different areas, such as software development, data analysis, product management, and quality assurance. We have made an effort to include as many free resources as possible.
```

Comment:
As we also have Randstad Housing as our resource, maybe we can adjust this line?

Something like:
"The resources collected here cover different areas, from tech-related, such as software development, data analysis, product management, and quality assurance, to the general ones that can help others, such as housing in The Netherlands."

We might want to remove the  "We have made an effort to include as many free resources as possible."
The reson is providing this repo already shows our effort to provide resources, and we have both free and paid resources.

Reply:
Thanks for these suggestions @adiati98 , I've used your text directly!